constants/geos_constants.fh

```diff
@@ -87,11 +99,15 @@ real(kind=RKIND), public, parameter :: AVOGNO   = 6.023000E+23_r8_kind      !< A
 real(kind=RKIND), public, parameter :: VONKARM  = 0.40_r8_kind              !< Von Karman constant [dimensionless]
 
 !--- Miscellaneous constants
-real(kind=RKIND), public, parameter :: ALOGMIN    = -50.0_r8_kind        !< Minimum value allowed as argument to log function [N/A]
-real(kind=RKIND), public, parameter :: EPSLN      = 1.0E-40_r8_kind      !< A small number to prevent divide by zero exceptions [N/A]
-real(kind=RKIND), public, parameter :: RADCON     = ((1.0E+02*GRAV)/(1.0D+04*CP_AIR))*SECONDS_PER_DAY  !< Factor to convert flux divergence
-                                                                                                !! to heating rate in degrees per day
-                                                                                                !! [deg sec/(cm day)]
-real(kind=RKIND), public, parameter :: RADCON_MKS = (GRAV/CP_AIR)*SECONDS_PER_DAY               !< Factor to convert flux divergence
-                                                                                                !! to heating rate in degrees per day
-                                                                                                !! [deg sec/(m day)]
+real(kind=RKIND), public, parameter :: ALOGMIN    = -50.0_r8_kind     !< Minimum value allowed as argument
+                                                                      !! to log function [N/A]
+real(kind=RKIND), public, parameter :: EPSLN      = 1.0E-40_r8_kind   !< A small number to prevent divide
+                                                                      !! by zero exceptions [N/A]
+real(kind=RKIND), public, parameter :: RADCON     = ((1.0E+02*GRAV)/(1.0D+04*CP_AIR))*SECONDS_PER_DAY
```

Comment:
The `!<` was moved to the next line but it needs to be on the same line as the variable its describing for it to get read by doxygen correctly.

If its close to the length limit you can start it with nothing, it'll still read the other lines:
```
real(kind=RKIND), public, parameter :: RADCON     = ((1.0E+02*GRAV)/(1.0D+04*CP_AIR))*SECONDS_PER_DAY !<
                                                                              !! rest of
                                                                              !! the comment
```